asset_dashboard/static/js/components/map_utils/MapZoom.js

```diff
@@ -0,0 +1,17 @@
+import React, { useEffect } from 'react'
+import { useMap } from 'react-leaflet'
+import zoomToSearchGeometries from './zoomToSearchGeometries'
+
+
+function MapZoom({ searchGeoms }) { 
+  const map = useMap()
+
+  useEffect(() => {
+    const group = new L.featureGroup()
+    zoomToSearchGeometries(map, group)
+  }, [searchGeoms])
```

Comment:
this will run whenever the incoming `searchGeoms` change

Reply:
Woah, cool trick!